pkg/visibility/api/rest/v1alpha1/test_utils.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package rest
+package v1alpha1
```

Comment:
This seems having nothing to do with the graduation, just a cleanup. Please send a separate PR for the cleanup.

Reply:
We need to keep both `v1alpha1` and `v1beta1`. To make it more clear I created two packages for each one.

Reply:
https://github.com/kubernetes-sigs/kueue/blob/b765d06caeae4c380f7f9555092e960ac6e31450/pkg/visibility/api/rest/v1alpha1/storage.go#L25-L32

Reply:
Right, but still this change (rename rest -> v1alpha1) could be done as a preparatory PR before the main PR?

Reply:
I can prepare another one.

Reply:
Done on https://github.com/kubernetes-sigs/kueue/pull/3041.